python/arcticdb/version_store/library.py

```diff
@@ -1255,6 +1255,46 @@ def get_description(self, symbol: str, as_of: Optional[AsOf] = None) -> SymbolDe
             date_range=info["date_range"],
         )
 
+    def get_description_batch(
+        self, symbols: List[Union[str, ReadRequest]], as_ofs: Optional[List[AsOf]] = None
```

Comment:
Change to use `ReadRequest` for the `as_ofs`

Reply:
done